<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addnote</class>
 <widget class="QDialog" name="addnote">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ajouter une fche de notation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>11</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="verticalSpacing">
      <number>15</number>
     </property>
     <item row="0" column="1">
      <widget class="QComboBox" name="trimestreComboBox">
       <item>
        <property name="text">
         <string>trimestre 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>trimestre 1</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="classesLabel">
       <property name="text">
        <string>classes</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="classesLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="disciplineLabel">
       <property name="text">
        <string>discipline</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="disciplineLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="annELabel">
       <property name="text">
        <string>ann√©e</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="annELineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="trimestreLabel">
       <property name="text">
        <string>trimestre</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="slugLabel">
       <property name="text">
        <string>slug</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="slugLineEdit">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>nom du slug (ce nom s'affichera pour identifier la fiche)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="minimumSize">
      <size>
       <width>382</width>
       <height>24</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addnote</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addnote</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
